### ISSUE TITLE
Function to remove regular patterns from Subject

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#47]
We need a function to remove patterns such as "Re: Fwd: [AW:] " etc. from the subject line. the ordinary `%header.set%`, `%header.prefix%` and `%header.append%` functions are not suitable. Since this is generally mainly useful for subject lines a dedicated function will be in order.

Syntax:
```
%header.deleteFromSubject(pattern)%
```

The patter should be wrapped in double quotes " ".
Example

```
%header.deleteFromSubject("([\[\(] *)?(Re|Fwd|AW?)*([-:;)\]][ :;\])-]*)|\]+")%
```
this will remove unwanted prefixes from the subject line

**Comment by RealRaven2000** [comment]
Tryout version:

[smartTemplate-fx-2.8.1pre30.zip](https://github.com/RealRaven2000/SmartTemplate4/files/4139414/smartTemplate-fx-2.8.1pre30.zip)

to install, unpack the contained file smartTemplate-fx-2.8.1pre30.xpi and drag into Add-ons manager.

**Comment by RealRaven2000** [comment]
was implemented in v2.9 on  09/02/2019
